File: adventofcode-2016/src/main/java/nl/mvdr/adventofcode/adventofcode2016/day14/Part2.java
package nl.mvdr.adventofcode.adventofcode2016.day14;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import nl.mvdr.adventofcode.solver.Solver;

public class Part2 implements Solver {

    private static final Logger LOGGER = LoggerFactory.getLogger(Part2.class);

    @Override
    public String solve(String inputfile) {
        return new OneTimePad(2017).solve(inputfile);
    }

    public static void main(String[] args) {
        Part2 instance = new Part2();

        String result = instance.solve("input-day14.txt");

        LOGGER.info(result);
    }
}

File: adventofcode-2025/src/main/kotlin/nl/mvdr/adventofcode/adventofcode2025/day05/Part2.kt
package nl.mvdr.adventofcode.adventofcode2025.day05

import io.github.oshai.kotlinlogging.KotlinLogging
import nl.mvdr.adventofcode.solver.FunctionSolver

private val logger = KotlinLogging.logger{}

fun solvePart2(linesSequence: Sequence<String>) = parseInput(linesSequence.toList()).countTotalFreshIds()

fun main() {
    val result = FunctionSolver(::solvePart2).solve("input-day05.txt")
    logger.info { result }
}
